app/forms/VATPaidFormProvider.scala

```diff
@@ -19,20 +19,35 @@ package forms
 import forms.Validation.monetaryPattern
 import javax.inject.Inject
 import forms.mappings.Mappings
+import models.RepaymentAmounts
 import play.api.data.Form
+import play.api.data.Forms.mapping
 
 class VATPaidFormProvider @Inject() extends Mappings {
 
-  def apply(): Form[String] =
+  def apply(): Form[RepaymentAmounts] =
     Form(
-      "value" -> decimal("vATPaid.error.required",
-        "vATPaid.error.notANumber")
-        .verifying(regexp(monetaryPattern, "vATPaid.error.decimalPlaces"))
-        .transform[BigDecimal](BigDecimal.apply, _.setScale(2).toString)
-        .verifying(maximumValue[BigDecimal](99999999999.99, "vATPaid.error.length"))
-        .transform[String](
-          d => d.toString,
-          i => i.toDouble
-        )
+      mapping(
+        "ActualPaidAmount" -> decimal("vatPaid.VATActuallyPaid.error.required",
```

Comment:
@simms11 could we keep these messages keys consistent? for the other pages I used 'actualamountpaid' and 'shouldhavepaid'